imdraw/imdraw.go

```diff
@@ -51,9 +51,11 @@ type IMDraw struct {
 	Precision int
```

Comment:
The pool can be done better. Here's my idea:

The pool should be a `[][]point`, just like now, but `poolindex` is unnecessary. At the beginning, the pool should be empty, no initialization needed. Then `getAndClearPoints` should return the **last** (think why) element of the pool, if available. if not available, it should return a new empty `[]point` slice. Then, add another method called, for example, `reuse(slice)`, which pushes the slice it obtains to the end of the pool. Call `imd.reuse(points)` at the end of each `polyline`, `fillEllipse` and so on methods. That'll do it.